aead/src/aead_key_templates.rs

```diff
@@ -112,7 +112,11 @@ pub fn x_cha_cha20_poly1305_key_template() -> KeyTemplate {
     }
 }
 
-/// Return a [`KeyTemplate`] that generates a KmsEnvelopeAead key for a given KEK in remote KMS
+/// Return a [`KeyTemplate`] that generates a `KmsEnvelopeAead key` for a given KEK in remote KMS.
```

Comment:
Is the backtick after key in the right place?

Reply:
Eagle-eyed spot!